doc/tutorial/07_vault.md

```diff
@@ -76,7 +76,7 @@ and press `Ctrl-d` (if running `cat`) to save the file.
 As the [**pgagroal**](https://github.com/agroal/pgagroal) operating system user, run the following command:
 
 ```
-pgagroal-admin -f /etc/pgagroal/pgagroal_vault_users.conf -U admin -P admin1234 add-user
+PGAGROAL_PASSWORD=admin1234 pgagroal-admin -f /etc/pgagroal/pgagroal_vault_users.conf -U admin user add
```

Comment:
Here I'm not sure if the original `add-user` was intentional, I could not find a working command for this, hence I changed it to `user add` whilst add it.

Reply:
Yes, correct